### DISCUSSION TITLE
Is there any function similar to excel function Sumproduct?

### DISCUSSION THREAD

**Discussion by happyych** [lifelib-dev/lifelib#63] [Q&A]
Is there any function similar to excel function Sumproduct?
I want to construct a model which has a lot of sumproduct calculation logic.

**Comment by fumitoh** [comment]
Here's an example of how to calculate the sum product of `foo(t)` and `bar(t)`  for `t = 0 .. last_t-1`.

```python
def sumproduct(t):
    return sum(foo(t) * bar(t) for t in range(last_t))
```

**Comment by happyych** [comment]
Thank you for your kind advise!
I will try the code right away


2024년 1월 14일 (일) 오후 10:53, fumitoh ***@***.***>님이 작성:
